MediaBrowser.Common/Net/IPNetAddress.cs

```diff
@@ -123,15 +123,27 @@ public static bool TryParse(string addr, out IPNetAddress ip)
                     if (IPAddress.TryParse(tokens[0], out res))
                     {
                         // Is the subnet part a cidr?
-                        if (byte.TryParse(tokens[1], out byte cidr))
+                        if (int.TryParse(tokens[1], out int cidr))
                         {
-                            ip = new IPNetAddress(res, cidr);
+                            if (cidr <= 0 || cidr >= 255)
```

Comment:
IPv4 has a maximum of /32 and IPv6 of /128
see https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_blocks

Reply:
in my defence - it was late at night.

Reply:
altered to 128

Reply:
This still accepts 192.168.2.1/128 which isn't valid

Reply:
But an IPv6 with /128 is still valid. Should be `> 128`

Reply:
Changed for each address type.